src/components/FormInput.jsx

```diff
@@ -1,13 +1,13 @@
 import styled from '@emotion/styled';
 
 export const FormInput = styled.input`
-  width: 268px;
+  width: 248px;
   max-width: 100%;
   height: 53px;
   border-radius: 7px;
   box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
```

Comment:
This RGB color is really working? It doesn't have "," between each "0", and it's declaring the opacity dividing the color between the opacity?

It doesn't should be some like:

```suggestion
  box-shadow: 2px 5px 6px rgba(0, 0, 0, 0.3);
```

instead?

Can you check please?

Reply:
done

Reply:
Thanks u man!